requirements.txt

```diff
@@ -20,3 +20,4 @@ aind-data-schema==0.36.0
 aind-data-schema-models==0.1.10
 pyyaml
 pyqtgraph
+stagewidget
```

Comment:
Do you also need to add it to the pyproject.toml?

Reply:
True, let me fix that real quick